### DISCUSSION TITLE
Jog controls not working properly

### DISCUSSION THREAD

**Discussion by Ivarsonk** [arkypita/LaserGRBL#1522] [Q&A]
I am completely new to laser engraving and all that. I have been messing around with it recently and i have noticed that i cannot move the x axis past the halfway point of my work area anymore unless i home the machine. Even then, if i move it to the right and then try to move it back to the left, the controls act as if i am still moving it to the right.

**Comment by jomo487** [comment]
I had a similar problem . check settings/ jog controls and uncheck continuous jog
